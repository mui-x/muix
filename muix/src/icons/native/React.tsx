import createSvgIcon from './lib/create-svg-icon'

export default createSvgIcon(
  'react',
  'React',
  true
)