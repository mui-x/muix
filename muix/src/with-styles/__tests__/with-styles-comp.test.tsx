/** @jsx Sheeter.createElement */
import React from 'react'

import * as Sheeter from 'reactxx-sheeter'

import { ts, traceComponentEx } from "reactxx-tests";

window.__TRACELEVEL__ = 2

describe("WITH STYLES", () => {
  it("WEB", () => {
    traceComponentEx(true,
      {
        root: ts.view = {
          backgroundColor: 'gray'
        }
      },
      ({ classes, classNameX, toClassNames, children }) => {
        const root = toClassNames([classes.root, classNameX])
        return <div classNameX={root}>
          {children}
        </div>
      },
      Comp => <Comp>
        Text
      </Comp>)
  });
});
