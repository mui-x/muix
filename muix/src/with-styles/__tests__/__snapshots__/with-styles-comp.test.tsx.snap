// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WITH STYLES WEB 1`] = `
<div
  className="a"
  trace={
    Array [
      ".a { background-color:gray /*root*/ }",
    ]
  }
>
  Text
</div>
`;
